website/util/metrics.py

```diff
@@ -28,6 +28,14 @@ def campaign_source_tag(campaign_name):
     return f'source:campaign|{campaign_name}'
 
 
+def unregistered_created_source_tag(reffer_id):
+    return f'source:unregistered_created|{reffer_id}'
```

Comment:
Similarly, a product or @brianjgeiger question, what do we expect the tags to be for unregistered contributors?